### PR TITLE
new icons: JetBrains Products

### PR DESCRIPTION
**Pull request by Kek5chen**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [ ] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes ä1125**

## Link to prove your SVG is correct and up-to-date.

https://www.jetbrains.com/de-de/company/brand/#logos-and-icons

### CODE REVIEW

**Comment by Snailedlt** [comment]
The icons look good, but at first glance I see one thing that needs to be changed, and one that should be added.

First off, you need to base your branch on the develop branch, and not master. And then when you open a PR, it should be for the develop branch.

2nd, it would be nice if you could add plain versions of the icons. That is, single color versions. Iirc, there should be some plain verions available on their website as well.

**Comment by Snailedlt** [comment]
Could you also add 
`This PR closes #1182` to the description of this PR?

File: icons/intellij/intellij-original.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...intellij-idea_intellij-idea_svg__b&quot;)" d="M70 18.6666L55.5 33L37 15L48.0666 1.16663L70 18.6666Z"></path><path fill="url(&quot;#intellij-idea_intellij-idea_svg__c&quot;)" d="M27.0667 60.4333L5.6 68.3667L10.0333 52.5L15.8667 33.1333L0 27.7667L10.0333 0L33.1333 2.8L54.5 31L55.5 33L27.0667 60.4333Z"></path><path fill="#000" d="M56 14H14V56H56V14Z"></path><g fill="#FFFFFF"><path d="M27.1366 22.1433V19.25H19.2733V22.1433H21.4666V32.1067H19.2733V34.9767H27.1366V32.1067H24.92V22.1433H27.1366Z" fill="#FFFFFF"></path><path d="M34.6967 35.21C33.46 35.21 32.4334 34.9767 31.6167 34.51C30.7767 34.0433 30.1 33.4833 29.5634 32.8533L31.7334 30.4267C32.1767 30.9167 32.6434 31.3133 33.0867 31.5933C33.5534 31.8733 34.0434 32.0133 34.6034 32.0133C35.2567 32.0133 35.77 31.8033 36.1434 31.3833C36.5167 30.9633 36.7034 30.31 36.7034 29.4V19.2733H40.25V29.5633C40.25 30.4967 40.1334 31.3133 39.8767 32.0133C39.62 32.7133 39.2467 33.2967 38.78 33.7633C38.29 34.2533 37.7067 34.6033 37.0067 34.86C36.3067 35.0933 35.5367 35.21 34.6967 35.21Z" fill="#FFFFFF"></path><path d="M34.4166 48.6499H18.6666V51.3332H34.4166V48.6499Z" fill="#FFFFFF"></path></g></symbol></defs><use xlink:href="#intellij-idea"></use></svg>
```
Comment (Snailedlt) [inline icons/intellij/intellij-original.svg:1]: [Viewbox should be 0 0 128 128](https://github.com/devicons/devicon/wiki/SVG-Standards)

Comment (Snailedlt) [reply icons/intellij/intellij-original.svg:1]: The same applies to all the icons you submitted^

**Comment by amacado** [comment]
Please "**Create a separate pull request (PR) for each Icon**" (see https://github.com/devicons/devicon/wiki/Example-of-Submitting-Icons-to-the-Repository). This is required for our automated workflow to work properly! :) Thank you!

**Comment by Kek5chen** [comment]
I mean. I like to help, but all I wanted was to use these icons in my profile. I'm not a visual designer or anything so I most likely will not assist you in this any further. You can gladly use the icons from my repo or the website i provided to update the existing jetbrains icons, but this is just too much work for my already filled schedule.

**Comment by Snailedlt** [comment]
@Kek5chen 
That's fine. I don't think most of the contributors here are visual designers, so if the reason is you feel you have a lack of skill, don't worry, you don't! :) But if the reason is (and it seems to be that way) that you don't have enough time, then all we can do is thank you for the time you put in so far, and good luck in your future endeavors. I'm sure someone will continue on this in not too long, so you can get your icons from devicon ;)

**Comment by Kek5chen** [comment]
i'll see what i can do. if i have some spare time on hand ill fix the stuff you suggested. it does not seem like its a huge thing to do

**Comment by Snailedlt** [comment]
@Kek5chen 
Sounds good to me!
Let us know if you need any help underways :)
I'd be willing to work on it together if you want

**Pull request closed without merging**